package.json

```diff
@@ -29,17 +29,34 @@
     "lint:js": "wp-scripts lint-js"
   },
   "devDependencies": {
+    "@types/react": "^18.2.47",
+    "@types/react-dom": "^18.2.18",
+    "@types/wordpress__blocks": "^12.5.18",
+    "@wordpress/block-editor": "^15.7.0",
+    "@wordpress/components": "^30.7.0",
+    "@wordpress/compose": "^7.34.0",
+    "@wordpress/data": "^10.34.0",
+    "@wordpress/editor": "^14.34.0",
+    "@wordpress/element": "^6.34.0",
```

Comment:
Why are these in `devDependencies`? I know they eventually are served by whatever WordPress Core has for them, but technically I think they're considered dependencies for the production code? Same for a few other `@wordpress/*` dependencies below.

Or the counter question would be: Why do you have only `@wordpress/icons` as a non-dev dependency?

Reply:
Moved now: https://github.com/WordPress/ai/pull/83/commits/eb8f5233c0c0e0c7d58183bf35e9fa058c0ebc5c